### ISSUE TITLE
New media type

### ISSUE THREAD

**Issue by arvislacis** [skripteria/wn-snowflake-plugin#19]
There could be a new Snowflake type `media`, or even maybe two new types - `mediaimage` and `mediafile`. At the core level they would use this: https://wintercms.com/docs/backend/forms#widget-mediafinder

^ I think some users would like to use existing images, files via Media manager rather than uploading new image, file. Or maybe users simply like the Media manager interface...

**Comment by skripteria** [comment]
Sound great!
Do you want to implement it?

**Comment by arvislacis** [comment]
Yes, I can implement this but currently this can be on hold while I found time for development. Anyways, what would be your preferable way of doing this - introducing just one `media` type or two new types - `mediaimage` and `mediafile`? Probably two type method would be better for specifying if only images or files are accepted for field...

**Comment by skripteria** [comment]
I think we shoud go for the 2 types, at leas this would be easier for the content preview I think.
I do have some spare time tomorrow so I can look at this one so you dont have to do everything alone

**Comment by arvislacis** [comment]
Ok, give it a try, should not be anything too hard - basically it's similar to already existing file and image types. The only difference is that media stores paths to the files in media manager and they are not attached to model directly.

**Comment by skripteria** [comment]
So far everything looks good, I am only stuck make the Image resizer work with a media image. I have asked for help in Discord, maybe somebody knows the way.
There is still the option to scale the original image down with CSS, but this may slow down the generation of the list view occasionally.

**Comment by skripteria** [comment]
First version is now working, couldnt find any problems when I tested it.
Maybe a few more tests and when we are happy with it I can publish it as v1.0.3

**Comment by skripteria** [comment]
README.md needs reflect the changes too, I will take care of it.

**Comment by arvislacis** [comment]
Ok... That's nice. Overall looks fine for me.

**Comment by skripteria** [comment]
v1.0.3 is released.
